src/schemas/lwm2m-types-versions.spec.ts

```diff
@@ -0,0 +1,45 @@
+import {
+	ConnectivityMonitoring_4_urn,
+	Device_3_urn,
+	Humidity_3304_urn,
+	Location_6_urn,
+	Pressure_3323_urn,
+	Temperature_3303_urn,
+} from './index.js'
+
+import { describe, test } from 'node:test'
+import assert from 'node:assert'
+import { parseURN } from '@nordicsemiconductor/lwm2m-types'
+
+/**
+ * The following versions are expected to be used by the LwM2M objects:
+ *
+ * | Object ID | Object Version | LwM2M Version |
+ * | --------- | -------------- | ------------- |
+ * | 3         | 1.2            | 1.1           |
+ * | 4         | 1.3            | 1.1           |
+ * | 6         | 1.0            | 1.0           |
+ * | 3303      | 1.1            | 1.0           |
+ * | 3304      | 1.1            | 1.0           |
+ * | 3323      | 1.1            | 1.0           |
+ *
+ * Failure to follow these versions could lead to errors.
```

Comment:
Explain this hypothesis.

Reply:
It is difficult to explain every potential issue the failure to follow the recommended versions could create. What I wanted to point out is that I guarantee that with those versions the converter will work but with other versions it could or could not. We already know one case where using different versions of the recommended version creates an unexpected behavior. 

However, by removing the duplication of information, that line was removed as well. 49dbbce3b42466ee5e749d7f309cccd16697ec8d